droopler.install

```diff
@@ -1666,3 +1666,44 @@ function droopler_update_8123(&$sandbox) {
     \Drupal::messenger()->addMessage($sandbox['current'] . ' paragraphs processed, ' . $sandbox['changed'] . ' changed.');
   }
 }
+
+/**
+ * Droopler 2.2 - remove "mt-5" spacing from CTA field in Reference Content Paragraph.
+ */
+function droopler_update_8124() {
+  /** @var Drupal\d_update\Updater $updater */
+  $updater = \Drupal::service('d_update');
+
+  $values['droopler__droopler_update_8124_1'] = update_modules_configuration(['d_p_reference_content'], '8124');
+  $updater->checklist()->saveProgress(['v2.2' => $values]);
+}
+
+/**
+ * Droopler 2.2 - remove "mt-5" spacing from CTA field saved in database.
+ */
+function droopler_update_8125(&$sandbox) {
+  $limit = 25;
+  $entities = \Drupal::entityQuery('paragraph')->condition('type', 'd_p_reference_content')->execute();
+  if (!isset($sandbox['current'])) {
+    $sandbox['current'] = 0;
+    $sandbox['max'] = count($entities);
+  }
+  $batch = array_slice($entities, $sandbox['current'], $limit);
+  foreach ($batch as $entity_id) {
+    $entity = \Drupal::entityTypeManager()->getStorage('paragraph')->load($entity_id);
+    if ($entity->hasField('field_d_cta_link')) {
+      $fields = $entity->get('field_d_cta_link');
+      foreach ($fields as $field) {
+        $value = $field->getValue();
+        // Check if field has a class 'mt-5'. If so, we remove it.
+        if ($key = array_search('mt-5', $value['options']['attributes']['class'])) {
```

Comment:
czyli `$value['options']['attributes']['class']` to arrayka, gdzie każdy element to pojedyńcza klasa ?

Reply:
No tak